lib/routes/dev/developers_view.dart

```diff
@@ -72,6 +75,8 @@ class DevelopersView extends StatelessWidget {
                 icon: Icons.share,
                 function: (_) => shareLog(),
               ),
+              Choice(
+                  title: "Export static backup", icon: Icons.charging_station, function: _exportStaticBackup)
```

Comment:
texts.backup_export_static

Reply:
Thank you, but do we want this to be translated? I see that it can cause confusion in costumer support cases where users have selected other languages. We do not use translated texts for this page in [Breezmobile](https://github.com/breez/breezmobile/blob/100160f3f45ac32ea705e252f148aa39fee28168/lib/routes/dev/dev.dart#L449-L460) either. Though I am not sure if this is a deliberate design choice.  
What do you think @roeierez ?

Reply:
You are right @ubbabeck please disconsider what I said before.